### PR TITLE
provide short name for describe as desc

### LINKED ISSUE #67305: Short name for describe
Feature request:

Can you please provide short name for `describe` as `desc`? This will save 4 less letters to type in many `kubectl` commands? 

I already use alias `k` for `kubectl`, `kc` for `kubectl -n kube-system` and `ki` for `kubectl -n istio-system`. For people like me who type with 2 fingers, short names are very helpful.

Thank you.

### PR DESCRIPTION
**Pull request by SataQiu**
**What this PR does / why we need it**:
  provide short name for describe as desc

**Which issue(s) this PR fixes**
  Fixes #67305

**Release note**:
```release-note
NONE
```

/sig cli
/kind feature

### CODE REVIEW

**Comment by SataQiu** [comment]
cncf-cla

File: pkg/kubectl/cmd/describe.go [hunk 1]
```diff
@@ -102,6 +102,7 @@ func NewCmdDescribe(parent string, f cmdutil.Factory, streams genericclioptions.
 
 	cmd := &cobra.Command{
 		Use: "describe (-f FILENAME | TYPE [NAME_PREFIX | -l label] | TYPE/NAME)",
+		Aliases: []string{"desc"},
```
Comment (nikhita) [inline pkg/kubectl/cmd/describe.go]: @SataQiu thanks for your first contribution to kubernetes!

This change lgtm, but it first needs to be confirmed by someone from sig-cli if this is ok to add.
/cc @soltysh @apelisse @seans3

Comment (dixudx) [reply pkg/kubectl/cmd/describe.go]: Currently we don't use aliases for subcommands. We use aliases for resources instead.

Comment (soltysh) [reply pkg/kubectl/cmd/describe.go]: I agree with what was said before. I don't want to introduce additional layer of confusion as we refactor the code. So no aliases (for now).

**Review by soltysh — CHANGES_REQUESTED** [review]

**Pull request closed without merging**